src/rtctools_heat_network/electricity_physics_mixin.py

```diff
@@ -245,6 +249,67 @@ def __electricity_cable_heat_mixing_path_constraints(self, ensemble_member):
 
         return constraints
 
+    def __voltage_loss_path_constraints(self, ensemble_member):
+        """
+        Furthermore, the voltage_loss symbol is set, as it depends on the chosen pipe
+        class, e.g. the related resistance and the current through the cable.
+
+        Parameters
+        ----------
+        ensemble_member : The ensemble of the optimization
+
+        Returns
+        -------
+        list of the added constraints
+        """
+        constraints = []
+        parameters = self.parameters(ensemble_member)
+
+        for cable in self.heat_network_components.get("electricity_cable", []):
+            cable_classes = []
+
+            current = self.state(f"{cable}.ElectricityIn.I")
+            v_loss = self.state(f"{cable}.V_loss")
+            r = parameters[f"{cable}.r"]
+            # v_loss_nom = parameters[f"{cable}.nominal_voltage_loss"]
+            v_nom = parameters[f"{cable}.nominal_voltage"]
+            c_length = parameters[f"{cable}.length"]
+
+            constraint_nominal = self.variable_nominal(v_loss)
+
+            # TODO: still have to check for proper scaling
+            try:
+                cable_classes = self._electricity_cable_topo_cable_class_map[cable]
+                variables = {
+                    cc.name: self.variable(var_name) for cc, var_name in cable_classes.items()
+                }
+                resistances = {cc.name: cc.resistance for cc in cable_classes}
+
+                big_m = v_nom
```

Comment:
I think we should do the big_m inside the for loop to compute it specific for each cable class

Reply:
You actually want a big_m that can cover all cable classes, in case that the size is not selected, big_m should be bigger than de v_loss - v_loss_cable size.

Reply:
ah ja off course, but let's use some upper bound of the variable based on max_resistance*max_current**2

Reply:
I'll think about a better upper bound. But max_resistance*max_current**2 might not be good enough because it should also overcome the voltage difference that might occur between two (disconnected) nodes due no cable being placed (DN0).